### PR TITLE
Fix styling issues / Add condition to show Github's profile if email isn't public

### PR DESCRIPTION
**Pull request by vaporwavie**
Hey @eveningkid! I've been working around this project of yours, gitfolio. 

This pull request fixes a few styling issues on SCSS. Some of the code on it was **deprecated**, so I fixed it. Also, **Github now hides the user's email if it isn't publicly configured** (I think it haves some _flag_ to enable it on the user dashboard). Well, that's basically it! Hope it helps. Great code of yours, man :smile: 

Oh, and I changed a few things on the README, but nothing big, just indentation :D

### CODE REVIEW

File: src/components/ReachMe.js [hunk 1]
```diff
@@ -1,45 +1,52 @@
...
+      )}
 
-      <span>
-        {
-          hireable ? (
+      {email !== null ? (
+        <span>
+          {hireable ? (
             <span>
               plus, I'm now <a href={mailtoAddress}>hireable</a>
```
Comment (eveningkid) [inline src/components/ReachMe.js]: Actually, we could do the same here:
```jsx
<span>
  {hireable ? "plus I'm now" : "I could soon be"} <a href={mailtoAdress}>hireable</a>
</span>
```

File: src/components/ReachMe.js [hunk 2]
```diff
@@ -1,45 +1,52 @@
...
 
   return (
     <div className="ReachMe">
-      <a href={mailtoAddress}>
-        <button>
-          why not getting in touch?
-        </button>
-      </a>
+      {email == null ? (
+        <a href={profile}>
```
Comment (eveningkid) [inline src/components/ReachMe.js]:
```jsx
<a href={email === null ? profile : mailToAddress}>
```
This would be more concise! :)

**Review by eveningkid — CHANGES_REQUESTED** [review]
Thanks for the PR and your words (again). Please check my comments, I think it would make the code even more readable!

**Comment by vaporwavie** [comment]
Okay! Gonna refactor with the code you suggested :)

**Comment by vaporwavie** [comment]
@eveningkid I did the changes as requested :) https://github.com/eveningkid/gitfolio/pull/1/commits/2f528483edb1426e89af616fd54dc22320aefd69

**Review by eveningkid — APPROVED** [review]

**Pull request merged**